webui/react/src/components/CompareParallelCoordinates.tsx

```diff
@@ -90,17 +89,17 @@ const CompareParallelCoordinates: React.FC<Props> = ({
       style: {
         axes: { label: { placement: 'after' } },
         data: {
-          series: chartData?.trialIds.map((trial) => ({
+          series: chartData?.recordIds.map((id) => ({
             lineWidth: 1,
-            strokeStyle: colorMap[trial],
+            strokeStyle: colorMap[id],
           })),
           targetDimensionKey: selectedMetric ? metricToStr(selectedMetric) : '',
         },
         dimension: { label: { angle: Math.PI / 4, truncate: 24 } },
         padding: [4, 120, 4, 16],
       },
     }),
-    [colorMap, hermesCreatedFilters, selectedMetric, chartData?.trialIds],
+    [colorMap, hermesCreatedFilters, selectedMetric, chartData?.recordIds],
   );
 
   const dimensions = useMemo(() => {
```

Comment:
question: not necessarily related to your changes, but is `dimensions` working as intended with Runs + Categorical/Log HP types? I think this is dependent on `experimentHyperparameters` having `selectedExperiments`, but I could be missing something.

Reply:
That's a good idea for future work! Right now runs api return hyperparameters as `Record<string, HpValue>`, so there's no hp type to do anything with.